Cargo.toml

```diff
@@ -14,7 +14,11 @@ maintenance = { status = "passively-maintained" }
 
 [features]
 raw = ["hashbrown", "hashbrown/raw"]
+static = ["hashbrown"]
 
 [dependencies]
 parking_lot = { version = "0.12.1", optional = true, default-features = false }
 hashbrown = { version = "0.12.2", optional = true, default-features = false, features = ["raw"] }
```

Comment:
... why do I always enable hashbrown's raw feature and then explicitly enable it again in our raw feature? 🤦

Reply:
Resolved in f516a683df84ddd50369015dd9b933ccda8c50e1